scripts/balancing.lua

```diff
@@ -1980,17 +1980,17 @@ DefineAnimations(
 )
 
 for i,spec in ipairs({
-   { Var = "Supply", Gold = 700, Unit = "brigand" },
-   { Var = "Demand", Gold = 1200, Unit = "ogre" },
+   { Var = "Supply", Gold = 1000, Wood = 4000, Unit = "brigand" }, --wood is used for cooldown. So long to reduce micro.
+   { Var = "Demand", Gold = 1200, Wood = 6000, Unit = "ogre" }, -- 1000 cooldown time = 200 production time (footman)
 }) do
    DefineSpell("spell-hire-" .. spec.Unit,
-      "manacost", 0, "range", 0, "target", "self", "cooldown", 5, "action", {
+      "manacost", 0, "range", 0, "target", "self", "cooldown", spec.Wood, "action", {
```

Comment:
I would not use cooldown for the spell like this. Leave it at `5`, that's just so you don't spam them out. The cooldown is really just modeling "I put my head in the door and ask if someone here would like a job". Asking takes a few seconds, so that's the `5` cooldown.